contracts/andromeda_auction/src/contract.rs

```diff
@@ -612,6 +631,16 @@ fn query_owner_of(
     Ok(res)
 }
 
+fn query_rates(deps: Deps) -> Result<Vec<RateInfo>, ContractError> {
+    let rates_opt = AUCTION_RATES.may_load(deps.storage)?;
+
+    if let Some(rates) = rates_opt {
+        Ok(rates)
+    } else {
+        Ok(vec![])
+    }
```

Comment:
I think this can be simplified to `AUCTION_RATES.may_load(deps.storage)?.unwrap_or_default()`

Reply:
Yep, that's correct. I'll adjust this.